cardano-simple/src/Cardano/Simple/PlutusLedgerApi/V1/Scripts.hs

```diff
@@ -78,6 +83,32 @@ mkStakeValidatorScriptPlutarch :: Config -> ClosedTerm a -> Either Text StakeVal
 mkStakeValidatorScriptPlutarch conf term =
   StakeValidator . Script . Plutarch.unScript <$> compile conf term
 
+mkPlutarchTypedScript :: Config -> ClosedTerm a -> Either Text (PlutarchTypedScript a)
```

Comment:
If you could add this error to the module level Error that would be great.

Reply:
I can't see any error level modules. There is just Error Text.